ci/detect-changes.py

```diff
@@ -78,7 +78,17 @@ def list_travis_affected_files():
     """
     Return a list of files affected in the current Travis build.
     """
-    commit_range = get_travis_commit_range()
+    if os.environ['TRAVIS_EVENT_TYPE'] == 'pull_request':
```

Comment:
Can you move this logic into `get_travis_commit_range`?

Reply:
Unless I'm misreading the diff, it looks like this logic has now been added to `get_travis_commit_range` and can be removed from here.

Reply:
Er, right, it looks like my mind was distracted when doing this.